api/endpoint/token.go

```diff
@@ -32,3 +34,65 @@ func GetTokensByAddress(c *gin.Context, tokenAPI blockatlas.TokenAPI) {
 	}
 	c.JSON(http.StatusOK, blockatlas.DocsResponse{Docs: &result})
 }
+
+// @Description Get tokens
+// @ID tokens_v3
+// @Summary Get list of tokens by map: coin -> [addresses]
+// @Accept json
+// @Produce json
+// @Tags Transactions
+// @Param data body string true "Payload" default({"60": ["0xb3624367b1ab37daef42e1a3a2ced012359659b0"]})
+// @Success 200 {object} blockatlas.ResultsResponse
+// @Router /v3/tokens [post]
+func GetTokens(c *gin.Context, tokensAPIs map[uint]blockatlas.TokenAPI) {
+	var query map[string][]string
+	if err := c.Bind(&query); err != nil {
+		c.JSON(http.StatusInternalServerError, model.CreateErrorResponse(model.InternalFail, err))
+		return
+	}
+	result := make(blockatlas.TokenPage, 0)
+	for coinStr, addresses := range query {
+		coinNum, err := strconv.ParseUint(coinStr, 10, 32)
```

Comment:
line 55 should already return `uint` value ?

Reply:
`uint64`
but coin in map is `uint`